guides/blending/client-side-blending/README.md

```diff
@@ -0,0 +1,159 @@
+
+<div class="code-name">PetList layout</div>
+
+```html 
+<style>
+    @import '/PetList/style.css';
+</style>
+<div class="pet-list-wrapper">
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-name"></slot>
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+```
+
+<div class="code-name">MedicalProvider layout</div>
+
+```html
+<slot name="medicalrecord/records-list-headline"></slot>
+<slot name="medicalrecord/records-list-table"></slot>
+```
+
+When these two layouts are merged with server-side blending, the resulting layout looks like this:
+
+```html
+<style>
+    @import '/PetList/style.css';
+</style>
+<div class="pet-list-wrapper">
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-name"></slot>
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+<slot name="medicalrecord/records-list-headline"></slot>
+<slot name="medicalrecord/records-list-table"></slot>
+```
+
+The layout from MedicalRecord is appended at the end of the PetList wrapper. To blend it, we will move the MedicalRecord table and headline into the `div class="pet-list-wrapper"` and expand the width of the wrapper to fit the table:
+
+```html
+<style>
+    @import '/PetList/style.css';
+    .pet-list-wrapper {
+        max-width: 750px;
+    }
+</style>
+<div class="pet-list-wrapper">
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-name"></slot>
+        <slot name="petlist/details-age-and-animal"></slot>
+    </div>
+    <div class="pet-list-wrapper__row">
+        <slot name="petlist/details-owner-name"></slot>
+        <slot name="petlist/details-weight"></slot>
+    </div>
+    <slot name="medicalrecord/records-list-headline"></slot>
+    <slot name="medicalrecord/records-list-table"></slot>
+    <slot name="petlist/details-list-link"></slot>
+</div>
+```
+
+We have now produced the result shown in the image above; the view from the MedicalRecord app has been neatly integrated with the view from PetList.
+
+## Creating Layouts
+
+The tool for creating these layouts is the [CompositionEditor](https://github.com/starcounterapps/CompositionEditor). It's a Starcounter app that can be opened at any page with `Ctrl + E`. The app in itself is an HTML editor that can edit the Shadow DOM layout. There are two main concepts in the CompositionEditor - the identifier and layout. 
+
+![Composition editor](/assets/CompositionEditor.png)
```

Comment:
This image does not render on https://github.com/Starcounter/Docs/blob/dbd0ad949fdd2d8d219dfcb57434746e51968a5b/guides/blending/client-side-blending/README.md